transformers/component_tests/queries/historical_ilk_state_computed_columns_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("current ilk state computed columns", func() {
+var _ = Describe("ilk_snapshot computed columns", func() {
```

Comment:
maybe worth also renaming the file?

Reply:
🤦‍♂ good call! I had a feeling I was forgetting something